packages/nimble-components/src/theme-provider/design-token-comments.ts

```diff
@@ -293,5 +293,29 @@ export const comments: { readonly [key in TokenName]: string } = {
     mentionFontFamily: 'Font family for mention views',
     mentionFontSize: 'Font size for mention views',
     mentionFontWeight: 'Font weight for mention views',
-    mentionFontLineHeight: 'Font line height for mention views'
+    mentionFontLineHeight: 'Font line height for mention views',
+    calendarStaticEventBackgroundColor:
```

Comment:
Can you give screenshots of static event, dynamic event, and transient event? I don't have intuition of those event types

Reply:
Added here: https://github.com/ni/nimble/pull/2310#discussion_r1698434728